pages/common/ugrep.md

```diff
@@ -0,0 +1,37 @@
+# ugrep
+
+> Ultra fast search tool with query UI, fuzzy search and more.
+> Try also `ugrep --help {{something}}`.
+> More information: <https://github.com/Genivia/ugrep>.
+
+- Start a [q]uery TUI to search files in the working directory recursively (CTRL-Z for help):
+
+`ugrep -Q`
+
+- Search the working directory recursively for files containing a regex pattern:
+
+`ugrep "{{pattern}}"`
+
+- Search for a case-[i]nsensitive regex pattern:
+
+`ugrep -i "{{pattern}}"`
```

Comment:
```suggestion
`ugrep --ignore-case "{{pattern}}"`
```

Reply:
No please don't use the long option here. The option `-I` is be the same as standard GNU grep, which has the short `-i` (and long one too). Otherwise the relationship of ugrep to grep is lost. Also other tools in tldr use short options like `-I` to mean the same thing, so why change this for ugrep?

Reply:
Let's keep the short option here to maintain compatibility.

Reply:
I agree. Not to be pedantic about this, but I question the practice to use long options with tldr pages. Isn't the goal to keep things simple? Long options require more mental effort and a lot more keyboard typing to execute. The names of short options (in the case of grep at least) are carefully decided in the past when these utilities were first introduced. Some options have a global meaning that is applicable to many other utilities as well (e.g. `-r` for recursion).